xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinExchange.java

```diff
@@ -17,12 +24,14 @@
    */
   public static final String PARAM_SANDBOX = "Use_Sandbox";
 
-  static final String SANDBOX_HOST = "openapi-sandbox.kucoin.com";
+  static final String SANDBOX_HOST = APIConstants.API_SANDBOX_HOST;
```

Comment:
don't think having two constants with the same value is necessary - would be best to get rid of one

Reply:
So would you would prefer to just harcode the strings in the KucoinExchange class and get rid of the entries in the APIConstants? This seems to be the approach in the other exchange modules, so I think you're right. I'll make the change.